packages/apm/src/integrations/tracing.ts

```diff
@@ -175,9 +175,9 @@ export class Tracing implements Integration {
       logger.warn(`[Tracing] We added a reasonable default for you: ${defaultTracingOrigins}`);
     }
 
-    // Starting our inital pageload transaction
+    // Starting pageload transaction
```

Comment:
Because in theory it can be used for other things besides just page load, and "idle" because it closes on idle timeout. I agree, though, that in the API review we're about to do, I'd like to reconsider the `startIdleTransaction` method name.